css-flexbox-abspos/index.html

```diff
@@ -0,0 +1,32 @@
+---
+feature_name: New CSS Flexbox behavior for absolute-positioned children
+chrome_version: 52
+feature_id: 6600926009753600
+---
+
+<h3>Background</h3>
+<p>A previous version of the CSS Flexible Box Layout specification set the static position of absolute-positioned children as though they were a 0x0 flex item. The latest version of the spec takes them fully out of flow and sets the static position based on align and justify properties.</p>
+
+<p>The sample CSS and HTML below demonstrate this. Starting with Chrome 52 or later, the green box will be perfectly centered in the red box. In other browsers, the top left corner of the green box will be in the top center of the red box.</p>
+
+{% capture css %}
+/*  */
+.container {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+.container > * {
+	position: absoulte;
+}
+{% endcapture %}
+{% include css_snippet.html css=css %}
+
+{% capture html %}
+<div class="container" style="background:red; width: 200px; height: 200px;">
```

Comment:
Does your mini samples framework have a way to apply style without it being visible in the example? Or, is there any other way to do this? These styles exist to make the result visible, but otherwise have nothing to do with the thing that was change. My intent was to show a CSS example without distractions.